### ISSUE TITLE
onbackpressed

### ISSUE THREAD

**Issue by croccio** [madcyph3r/AdvancedMaterialDrawer#2]
why if i press back exit app??? please make it customizable

**Comment by madcyph3r** [comment]
See back press pattern in the example. you can customize your back action.
Here is an example. https://github.com/madcyph3r/AdvancedMaterialDrawer/blob/master/example/src/main/java/de/madcyph3r/example/tools/BackPatternCustom.java

**Comment by croccio** [comment]
@madcyph3r i've override it but you in code set finish when i press section so main activity that has drawer will close when i open new activity from section

**Comment by madcyph3r** [comment]
if you open a new activity, you must set a new menu to this new activity. Otherwise you must use a fragment, then the menu will be there. This is normal. 
If you don't want the finish, then make your own onClickListener and start the activity in this method, without finish.

**Comment by croccio** [comment]
thank you very much. i've create my own onClickListener but i can't unselect previous section. in my activity i've create a second variable called oldSection but i think that is better if there was a method unselectPreviousSection(). now there is a similar method, but i must give it a section as parameter but i think that this will be transparent to the developer. can i fork your project and make a pull request?

**Comment by madcyph3r** [comment]
I have added a new Listener, before on section change and after on section change listener. In the before method you can call this.getCurrentSection().unSelect(); this will unselect the old section.
Example:

```
    @Override
    public void onBeforeChangedSection(MaterialSection newSection) {
        if(getCurrentSection() != newSection) {
            this.getCurrentSection().unSelect();
        }
    }
```

It's available in the  new release 1.0.7 . I will upload it in a few hours.
